### PR TITLE
Add type annotations

### PR DESCRIPTION
**Pull request by woxtu**
Hi,

This PR adds type annotations to fix some ESLint warnings related to `PyRepl`.

### CODE REVIEW

**Review by marimeireles — COMMENTED** [review]
Thank you for the PR!
Other than this question, it LGTM :)

File: pyscriptjs/src/components/pyrepl.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { basicSetup, EditorState, EditorView } from '@codemirror/basic-setup';
 import { python } from '@codemirror/lang-python';
-import { Compartment, StateCommand } from '@codemirror/state';
-import { keymap } from '@codemirror/view';
+import { Compartment } from '@codemirror/state';
+import { keymap, type Command } from '@codemirror/view';
```
[the remark this answers is not in the document]
Comment (woxtu) [reply pyscriptjs/src/components/pyrepl.ts:4]: Thank you for pointing that out. I notice my misreading the reference manual, so I'll revert that.

[the remark this answers is not in the document]
Comment (marimeireles) [reply pyscriptjs/src/components/pyrepl.ts]: That was my interpretation, but I'm a beginner in codemirror. Feel free to voice something else if you disagree =)

**Comment by marimeireles** [comment]
Thanks for adding types! :)

**Pull request merged**